packages/react/src/components/Select/Select.tsx

```diff
@@ -139,7 +130,7 @@ const Select = React.forwardRef<SelectElement, SelectProps>((props, forwardedRef
     >
       <div className={classes} ref={forwardedRef}>
         {startIcon && (
-          <span className={cx('manifest-select--icon', 'manifest-select--icon--start')}>
+          <span className={cx('manifest-select__icon', 'manifest-select__icon--start')}>
```

Comment:
why the change in classname? Is this due to the update?

Reply:
yeah it was a break from a previous commit